mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/MailboxMapper.java

```diff
@@ -100,4 +101,19 @@ boolean hasChildren(Mailbox<Id> mailbox, char delimiter)
      * @throws MailboxException 
      */
     List<Mailbox<Id>> list() throws MailboxException;
+
+    /*
+     * TODO: Those comment should be deteled whenever the ticket is approved
+     * 
+     * BEGIN: Added by Quynhnn for RFC-5464 
+     */
+    MailboxAnnotation getMetadata(Mailbox<Id> mailbox) throws MailboxException;
```

Comment:
getMetadata or getAnnotations?

I think we need a consistent vocabulary here...